src/tests/functional/plugin/shared/include/set_device_name.hpp

```diff
@@ -6,12 +6,12 @@
 
 #include <string>
 
-#include "common_test_utils/test_constants.hpp"
+#include "test_constants.hpp"
```

Comment:
Seems like this change should be reverted and in that case cmake update won't be needed

Reply:
Fixed.